src/components/TimeseriesChart/TimeseriesChart.tsx

```diff
@@ -203,11 +246,11 @@ export class TimeseriesChart extends React.Component<
                 }}
                 height={height}
                 width={width}
-                ruler={ruler}
+                ruler={{ visible: false }}
```

Comment:
Guess we can just remove ruler prop here

Reply:
Yes - don't know what I was thinking :)

Reply:
It is a required property on `LineChart` - so we need it here.